src/main/java/com/khadri/jdbc/prepare/data/student/processor/StudentDataProcessor.java

```diff
@@ -16,7 +17,7 @@ public StudentDataProcessor(Scanner scanner, StudentInsertDao insertDao) {
 		this.insertDao = insertDao;
 	}
 
-	public void process(int recordNumber) {
+	public void process(int recordNumber) throws ClassNotFoundException, SQLException {
```

Comment:
define private modifier.

Reply:
this method  called main class so kept only public